tests/unit/page/page_core.js

```diff
@@ -84,11 +84,11 @@ asyncTest( "page container is updated to page theme at pagebeforeshow", function
 
 	var pageTheme = "ui-overlay-" + $.mobile.activePage.page( "option", "theme" );
 
-	$.mobile.pageContainer.removeClass( pageTheme );
+	$( ".ui-pagecontainer" ).removeClass( pageTheme );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?